Sources/PaymentsCore/Networking/APIClient.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+public class APIClient {
```

Comment:
Would it be worth making this a final class?

Reply:
done!